doc/intro.rst

```diff
@@ -11,6 +11,12 @@ canvas, and other functionality to support:
 * color/greyscale (where supported),
 * dithering to monochrome
 
+**New**  The HD44780 (and similar) devices are some of the most popular small LCD displays
```

Comment:
I would remove the 'new' because it'll become outdated.

Reply:
Just a bit of marketing, but I agree.  It has been removed.